libs/barista-components/checkbox/src/_checkbox-theme.scss

```diff
@@ -1,40 +1,42 @@
-@import '../../core/src/style/variables';
-@import '../../core/src/theming/theming';
 @import '../../core/src/style/interactive-common';
+@import '../../core/src/style/variables';
 
-@mixin dt-theme-checkbox($theme) {
-  $is-dark: map-get($theme, 'is-dark');
+.dt-theme-dark {
+  :host {
+    .dt-checkbox-content {
+      color: #ffffff;
+    }
 
-  @if ($is-dark == true) {
-    :host {
-      .dt-checkbox-content {
-        color: #ffffff;
-      }
+    .dt-checkbox-container {
+      border-color: #ffffff;
+    }
+
+    .dt-checkbox-icon svg {
+      stroke: #ffffff;
+    }
 
+    &:hover {
       .dt-checkbox-container {
         border-color: #ffffff;
```

Comment:
The border-color is already set in L11

Reply:
Well, the styling isn't correct if I don't set that twice. It will use the usual hover styling as if it isn't in the dark theme.

Reply:
@rowa-audil is correct here, the base hover is overwritten by the checkbox.scss:56 rule. We can either live with the duplication, or group the selectors: 
```scss
:host .dt-checkbox-container,
:host:hover .dt-checkbox-container {
  boder-color: #ffffff;
}
```